polkadot/node/core/backing/src/lib.rs

```diff
@@ -1157,10 +1157,8 @@ async fn construct_per_relay_parent_state<Context>(
 		.await?
 		.unwrap_or(NodeFeatures::EMPTY);
 
-	let inject_core_index = node_features
-		.get(FeatureIndex::ElasticScalingMVP as usize)
-		.map(|b| *b)
-		.unwrap_or(false);
+	// elastic scaling on
+	let inject_core_index = true;
```

Comment:
why not remove this variable as well as all of its uses? (we can assume it's true and remove the code that handles the potential of it being false)

Reply:
Okay this shades a lot more light will start on that @alindima

Reply:
@alindima  Done